circuit_maintenance_parser/parser.py

```diff
@@ -161,3 +162,21 @@ def clean_line(line):
             line = line.strip()
         # TODO: below may not be needed if we use `quopri.decodestring()` on the initial email file?
         return line.replace("=C2", "").replace("=A0", "").replace("\r", "").replace("=", "").replace("\n", "")
+
+
+class EmailDateParser(Parser):
+    """Parser for Email Date."""
+
+    _data_types = ["email-header-date"]
+
+    def parse(self, raw: bytes) -> List[Dict]:
+        """Method that returns a list of Maintenance objects."""
```

Comment:
Does this method only support specific formats of dates?

Reply:
Only supports Email Date format